recipes/pbtk/build.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+mkdir -p "${PREFIX}"/bin
+
+cp ccs-kinetics-bystrandify "${PREFIX}"/bin/
+chmod +x "${PREFIX}"/bin/ccs-kinetics-bystrandify
+
+cp extracthifi "${PREFIX}"/bin/
+chmod +x "${PREFIX}"/bin/extracthifi
+
+cp pbindex "${PREFIX}"/bin/
+chmod +x "${PREFIX}"/bin/pbindex
+
+cp pbindexdump "${PREFIX}"/bin/
+chmod +x "${PREFIX}"/bin/pbindexdump
+
+cp pbmerge "${PREFIX}"/bin/
+chmod +x "${PREFIX}"/bin/pbmerge
+
+cp zmwfilter "${PREFIX}"/bin/
```

Comment:
copy all binaries, then at the end just do a `chmod +x "${PREFIX}"/bin/*`?

Reply:
You didn't do that for uny...maybe I don't want to chmod stuff that is in `"${PREFIX}"/bin/`  but does not belong to this package?